### PR TITLE
test display name for more datatypes

### PR DESCRIPTION
**Pull request by perolavsvendsen**
Solving #145

### CODE REVIEW

File: tests/test_export_item.py [hunk 1]
```diff
@@ -397,29 +397,66 @@ def test_display_name():
...
         config=CFG2,
         content="depth",
         verbosity="DEBUG",
     )
 
-    obj = xtgeo.RegularSurface(ncol=3, nrow=4, xinc=22, yinc=22, values=0)
-    exporter = ei._ExportItem(dataio, obj, verbosity="DEBUG")
+    grid = xtgeo.Grid()
+    grid.create_box()
+    grid.name = "ObjectName"
```
Comment (jcrivenaes) [inline tests/test_export_item.py:409]: Prefer grid = xtgeo.create_box_grid() syntax (ongoing change in xtgeo)

File: tests/test_export_item.py [hunk 2]
```diff
@@ -455,3 +492,66 @@ def test_display_name():
...
+    # when nothing received from grid object, return None
+    dataio = fmu.dataio.ExportData(
+        config=CFG2,
+        content="depth",
+        verbosity="DEBUG",
+    )
+
+    grid = xtgeo.Grid()
+    grid.create_box()
+
```
Comment (jcrivenaes) [inline tests/test_export_item.py:553]: `grid = xtgeo.create_box_grid()`

**Review by jcrivenaes — COMMENTED** [review]
Is `ObjectName` a good idea? The intention with `object.name` is that  most (shall be all) xtgeo objects will have an automatic `name` attribute which can be used

File: tests/test_export_item.py (hunk 2 above)
Comment (perolavsvendsen) [reply tests/test_export_item.py]: Done

File: tests/test_export_item.py (hunk 1 above)
Comment (perolavsvendsen) [reply tests/test_export_item.py]: Done

**Comment by perolavsvendsen** [comment]
> Is `ObjectName` a good idea? The intention with `object.name` is that most (shall be all) xtgeo objects will have an automatic `name` attribute which can be used

In what context...? As far as I know, this term is only used for variable name in the tests to point out that this is the name stored in the object itself (last fallback).

**Comment by perolavsvendsen** [comment]
Ref use of `xtgeo.create_box_grid()`. Latest version of xtgeo causes lots of warnings in the tests now, and probably requires a slightly larger cleanup. Will create separate issue on this.

**Review by jcrivenaes — APPROVED** [review]
🚀

**Pull request merged**